siliconcompiler/scheduler/scheduler.py

```diff
@@ -969,3 +972,25 @@ def __check_tool_versions(self) -> bool:
                 os.chdir(cwd)
 
         return not error
+
+    def mark_copy(self) -> bool:
+        """
+        Iterates through all tasks in the scheduler, retrieves the required
+        keys for each task, and marks those paths as "copy" in the project's
+        data structure.
+
+        Returns:
+            bool: True if the operation is successful, False otherwise.
+        """
+
+        try:
+            for _, task in self.__tasks.items():
+                keys = task.get_required_path_keys()
+
+                for key in keys:
+                    self.project.set(*key, True, field='copy')
```

Comment:
probably just move this to the scheduler node too.
This is only needed in the slurm scheduler, the others can skip this for now.